versioned_docs/version-1.13/reference/development-environments.mdx

```diff
@@ -22,7 +22,7 @@ Coming to the first step of deploying your application code, there are multiple
 
 A second option available is [Okteto SaaS](https://www.okteto.com/pricing/?plan=SaaS). Using Okteto SaaS, you don't have to take care of setting up the infrastructure for deployment yourself. Okteto SaaS was built keeping dev environments in mind, so it also comes with useful [additional features](index.md) you would not find in a traditional cluster. To know more about it, you can head over to the Okteto SaaS [documentation](index.md). 
 
-Another option available is [Okteto Self-Hosted]https://www.okteto.com(/pricing/). Okteto Self-Hosted gives you all the features of Okteto SaaS - on your own infrastructure. This offers you full control of your Kubernetes infrastructure while delivering the same smooth experience you expect from Okteto SaaS. The Okteto Self-Hosted [documentation](self-hosted.mdx) covers how you can get started with it.
+Another option available is [Okteto Self-Hosted](https://www.okteto.com/pricing/). Okteto Self-Hosted gives you all the features of Okteto SaaS - on your own infrastructure. This offers you full control of your Kubernetes infrastructure while delivering the same smooth experience you expect from Okteto SaaS. The Okteto Self-Hosted [documentation](self-hosted.mdx) covers how you can get started with it.
```

Comment:
Great catch @SD-13 , could you also fix it in this file?
https://github.com/okteto/docs/blob/main/src/content/reference/development-environments.mdx#L25

Reply:
@pchico83 Sure, I also found this broken link in older versions of doc for e.g. 1.10, 1.11 and 1.12. Do we care about that?

Reply:
Fixing current version and next is good enough, thanks!